recipes-graphics/mesa/mesa_%.bbappend

```diff
@@ -0,0 +1,22 @@
+# Remove the *.so libraries - the ones provided by mali-450 driver are to
```

Comment:
Commit message needs to say this adds a bbappend and that it only does it for hikey, the current form is too vague